vcs/__init__.py

```diff
@@ -189,6 +189,15 @@ class VCSDeprecationWarning(DeprecationWarning):
     ("default", "AvantGarde-Book_Bold.ttf"),
     ("Clarendon", "Clarendon.ttf"),
     ("Courier", "Courier.ttf"),
+    ("DejaVuSans-Bold","DejaVuSans-Bold.ttf"),
```

Comment:
@jccosta could you add the new fonts last, inserting them here will change the number for the fonts after them and if people rely on font number rather than name for their script it will mess them up.